prover/constraints-versions.txt

```diff
@@ -0,0 +1,2 @@
+  v0.8.0-rc7
```

Comment:
In the future, it would be great if `constraints-versions.txt` could be in `.json` or another format and include useful information, such as the commit of the constraint, etc.